Chap_API_Event.tex

```diff
@@ -473,14 +481,17 @@ \subsection{\code{PMIx_Notify_event}}
 
 If \refarg{cbfunc} is \code{NULL}, the function will be treated as a \emph{blocking} call and the result of the operation returned in the status code.
 
-If \refarg{cbfunc} is non-\code{NULL}, the function will be treated as a \emph{non-blocking} call and return one of the following:
+If \refarg{cbfunc} is non-\code{NULL}, the function will be treated as a \emph{non-blocking} call.
+
+\returnsimplenb
+Note that a successful call does \textit{not} reflect the success or failure of delivering the event to any recipients.
 
+\returnstart
 \begin{itemize}
-\item \refconst{PMIX_SUCCESS} The notification request is valid and is being processed. The callback function will be called when the process-local operation is complete and will provide the resulting status of that operation. Note that this does \textit{not} reflect the success or failure of delivering the event to any recipients. The callback function must not be executed prior to returning from the \ac{API}.
-\item \refconst{PMIX_OPERATION_SUCCEEDED}, indicating that the request was immediately processed and returned \textit{success} - the \refarg{cbfunc} will \textit{not} be called
+\item \refconst{PMIX_OPERATION_SUCCEEDED}, returned only when \refarg{cbfunc} is non-\code{NULL} and the request was immediately processed successfully - the \refarg{cbfunc} will \textit{not} be called
```

Comment:
As above - the `cbfunc` argument is ignored in this case.

Reply:
Added a revision exception change for this